internal/pkg/template/template.go

```diff
@@ -42,28 +44,30 @@ func LoadInputs(fs filesystem.Fs) (*Inputs, error) {
 }
 
 type dependencies interface {
-	Ctx() context.Context
 	Logger() log.Logger
 	StorageApi() (*storageapi.Api, error)
 	SchedulerApi() (*schedulerapi.Api, error)
 }
 
+type _reference = model.TemplateRef
```

Comment:
😱 prefixování podtržítkem je taky Go konvence? 😅

Reply:
asi nie :) potrebujem to spravit `private`, aby sa nedala nastavit hodnota zvonku + potrebujem aby to bolo embedded.

To sa standardne robi tak, ze sa vytvori alias.
No a dal som tam `_` aby sa mi to nemiesalo s premenymi s nazvom `reference`.